flowmap/subgrid.py

```diff
@@ -77,7 +77,7 @@ def build_interpolate(grid, values):
     return L
 
 
-def build_tables(grid, dem):
+def build_tables(grid, dem, id_grid):
```

Comment:
You're using this function also in formats.ugrid line 280. Didn't change the argument input